### PR TITLE
general apps definition

### PR DESCRIPTION
**Pull request by zikrurR**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]

File: ccd-definition/AuthorisationCaseField/systemUpdateGAspec.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseFieldID": "generalAppPBADetails",
+    "UserRole": "caseworker-civil-systemupdate",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
```
Comment (karthick-mohanasundaram-HMCTS) [inline ccd-definition/AuthorisationCaseField/systemUpdateGAspec.json:21]: There are some missing fields. eg. generalAppEvidenceDocument

Comment (karthick-mohanasundaram-HMCTS) [reply ccd-definition/AuthorisationCaseField/systemUpdateGAspec.json:21]: Please check in this PR - https://github.com/hmcts/civil-ccd-definition/pull/369/files

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec.json [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseFieldID": "generalAppStatementOfTruthConsent",
+    "UserRoles": [
+      "caseworker-civil-solicitor"
+    ],
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "CaseFieldID": "generalAppUIStatementOfTruth",
```
Comment (karthick-mohanasundaram-HMCTS) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec.json:77]: generalAppUIStatementOfTruth is been modified to different name. Please check the PR. https://github.com/hmcts/civil-ccd-definition/pull/369/files

**Comment by karthick-mohanasundaram-HMCTS** [comment]
@zikrurR Smitha has introduce some changes in naming convention in the PR - https://github.com/hmcts/civil-ccd-definition/pull/369/files. Please take a look.

File: ccd-definition/AuthorisationCaseField/systemUpdateGAspec.json (hunk 1 above)
Comment (zikrurR) [reply ccd-definition/AuthorisationCaseField/systemUpdateGAspec.json:21]: These will need to be added later by @smitatalawadekar-hmcts .

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec.json (hunk 2 above)
Comment (zikrurR) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec.json:77]: Same as stated earlier, what @smitatalawadekar-hmcts is working on is a work in progress. This work precedes what @smitatalawadekar-hmcts has done. After this is merged then she can take it and add the necessary modifications. I hope this makes it clear.

**Review by faranhmcts — APPROVED** [review]

**Pull request merged**